src/muxer/mplex/mplex_stream.cpp

```diff
@@ -135,9 +135,11 @@ namespace libp2p::connection {
     if (bytes == 0 || in.empty() || static_cast<size_t>(in.size()) < bytes) {
       return cb(Error::INVALID_ARGUMENT);
     }
+    std::vector<uint8_t> in_vector(in.size());
```

Comment:
```suggestion
    std::vector<uint8_t> in_vector{in.begin(), in.end()};
```

Reply:
`streamWrite` below accepts `span`, so this conversion to `vector` should be moved into `if (is_writing_)` block

Reply:
but is it safe to pass span into the streamWrite? Can this span keep no bytes when it should be used?